cli/ts/commands/proveOnChain.ts

```diff
@@ -214,11 +214,15 @@ export const proveOnChain = async ({
       logError("currentSbCommitment mismatch.");
     }
 
-    const coordPubKeyHashOnChain = BigInt(await pollContract.coordinatorPubKeyHash());
+    const [coordPubKeyHashOnChain, coordinatorPubKeyOnChain] = await Promise.all([
+      pollContract.coordinatorPubKeyHash(),
+      pollContract.coordinatorPubKey(),
+    ]);
+
     if (
       hashLeftRight(
-        BigInt((circuitInputs.coordPubKey as BigNumberish[])[0]),
-        BigInt((circuitInputs.coordPubKey as BigNumberish[])[1]),
+        BigInt((coordinatorPubKeyOnChain as BigNumberish[])[0]),
```

Comment:
should it be instead `if (circuitInputs.coordintorPubKeyHash !== coordPubKeyHashOnChain.toString())`